src/concrete/ml/sklearn/base.py

```diff
@@ -1410,10 +1433,13 @@ def post_processing(self, y_preds: numpy.ndarray) -> numpy.ndarray:
         # Sum all tree outputs
         # Remove the sum once we handle multi-precision circuits
         # FIXME: https://github.com/zama-ai/concrete-ml-internal/issues/451
-        y_preds = numpy.sum(y_preds, axis=-1)
+        if os.getenv("TREES_USE_FHE_SUM") == "0":
```

Comment:
the main problem I see with this sort of behavior is in a client-server setting : if the client's env variables are not matching the ones from the server, we'll have some issues here .. 

but I'm not able to find any proper solutions yet. I thought of a `self._run_sum_in_fhe = os.getenv("TREES_USE_FHE_SUM")` in the `__init__` for ex (which would fix the problem I mentionned) but I'm really not sure it's a good practice

Reply:
any thoughts on it @zama-ai/machine-learning

Reply:
Yes we should store this as an attribute in the object.

I'm not sure why we went for an env var and not a parameter in the object ?

Reply:
I think you are both right !

Reply:
The choice of the env variable was because this is not something we want people to use as it's more of an experimental feature. But now we want to make it compatible with the deployment. I find this a bit sketchy. If we want this to be an actual option then yes we should go for a parameter instead of a env variable. But the feature brought some performance loss in some cases thus why it's not the new default.